cmaes/_warm_start.py

```diff
@@ -0,0 +1,79 @@
+import math
+import numpy as np
+
+from typing import Tuple, List
+
+
+def get_starting_point(
+    source_solutions: List[Tuple[np.ndarray, float]],
+    gamma: float = 0.1,
+    alpha: float = 0.1,
+) -> Tuple[np.ndarray, float, np.ndarray]:
+    """Estimate a promising distribution of the source task, then
+    returns a starting point (mean vector and the covariance matrix
+    of the multivariate gaussian distribution in the CMA-ES).
```

Comment:
[nits] returns a multivariate gaussian distribution (the mean vector and the covariance matrix) used for initialization of the CMA-ES.